doc/en/user/source/community/jwt-headers/configuration.rst

```diff
@@ -0,0 +1,211 @@
+.. _community_jwtheaders_config:
+
+JWT Headers configuration
+=========================
+
+The JWT Headers module covers three main use cases;
+
+#. Simple Text, JSON, or JWT headers for the username
+#. Verification of JWT Access Tokens
+#. Getting roles from a JSON header or an attached JWT Access Token claim
+
+Configuration Options
+---------------------
+
+User Name Options
+^^^^^^^^^^^^^^^^^
+
+.. list-table:: User Name Options
+   :header-rows: 1
+
+   * - Config Option
+     - Meaning
+   * - Request header attribute for User Name
+     - The name of the HTTP header item that contains the user name.
+   * - Format the Header value is in
+     - Format that the user name is in:
+	 
+	   * Simple String - user name is the header's value.
```

Comment:
This behavior seems pretty similar to https://docs.geoserver.org/main/en/user/community/web-service-auth/configuration.html ?

Reply:
Sorry, I linked the wrong page... see this one: https://docs.geoserver.org/main/en/user/security/tutorials/httpheaderproxy/index.html

Reply:
Yes - there are a few modules where you can get the username from a simple text header (there's an extension for doing that).  The only reason I allow a simple "text only" username header is for completeness - I expect that people using this module would use the JSON or JWT (with JWT validation).

Second, its unclear (i didn't directly test them) if those other modules properly handle "logging out" the user if you have multiple authentication modules in the filter chain.  This one is much smarter about that kind of thing.

I can remove (or hide) that functionality if its a problem.